server/src/modules/notifications/agents/agent.ts

```diff
@@ -0,0 +1,24 @@
+import { Notification } from '../entities/notification.entities';
+import {
+  NotificationAgentConfig,
+  NotificationAgentKey,
+  NotificationType,
+} from '../notifications-interfaces';
+
+export interface NotificationPayload {
+  event?: string;
+  subject: string;
+  notifySystem: boolean;
+  image?: string;
+  message?: string;
+  extra?: { name: string; value: string }[];
```

Comment:
I think this is a leftover from overseerr as I can't see it being used.

Reply:
Indeed, also something that still needs cleaning up